### PR TITLE
Move commands from client to bundle, add ProveGssfPossessionCommand

### PR DESCRIPTION
**Pull request by DRvanR**
Depends on https://github.com/SURFnet/Stepup-Middleware/pull/46

### CODE REVIEW

**Comment by DRvanR** [comment]
@rjkip

File: src/Surfnet/StepupMiddlewareClient/Service/CommandService.php [hunk 1]
```diff
@@ -48,11 +48,11 @@ class CommandService
     public function __construct(ClientInterface $guzzleClient, $username, $password)
     {
         if (!is_string($username)) {
-            InvalidArgumentException::invalidType('string', 'username', $username);
+            throw InvalidArgumentException::invalidType('string', 'username', $username);
```
Comment (rjkip) [inline src/Surfnet/StepupMiddlewareClient/Service/CommandService.php]: Not the first time I've fallen into that trap. :sweat_smile:

File: src/Surfnet/StepupMiddlewareClient/Service/CommandService.php [hunk 2]
```diff
@@ -71,7 +71,6 @@ public function __construct(ClientInterface $guzzleClient, $username, $password)
     public function execute($commandName, $uuid, array $payload, array $metadata = [])
     {
         $this->assertIsValidCommandName($commandName);
-        InvalidArgumentException::invalidType('string', 'uuid', $uuid);
```
Comment (rjkip) [inline src/Surfnet/StepupMiddlewareClient/Service/CommandService.php]: For my information: you want to keep validation to the middleware?

**Comment by rjkip** [comment]
dfadfd

:+1:

**Pull request merged**